setup/osi_analytic_segments_defaults/odoo/addons/osi_analytic_segments_defaults

```diff
@@ -0,0 +1 @@
+../../../../osi_analytic_segments_defaults
```

Comment:
@Freni-OSI It looks like there are segment item changes in this PR, can you clean that up then this should be able to be reviewed.

Reply:
Done. Thanks